modules/ROOT/pages/provisioning-azure.adoc

```diff
@@ -0,0 +1,62 @@
+= Provisioning Fedora CoreOS on Azure
+
+This guide shows how to provision new Fedora CoreOS (FCOS) nodes on Azure.  Fedora currently does not publish Fedora CoreOS images within Azure, so you must download an Azure image from Fedora and upload it to your Azure subscription.
+
+== Prerequisites
+
+Before provisioning a FCOS machine, you must have an Ignition configuration file containing your customizations. If you do not have one, see xref:producing-ign.adoc[Producing an Ignition File].
+
+You also need to have access to an Azure subscription.
+
+== Downloading an Azure image
+
+Fedora CoreOS is designed to be updated automatically, with different schedules per stream.
+Once you have picked the relevant stream, download, verify, and decompress the latest Azure image:
+
+[source, bash]
+----
+export STREAM="stable"
+coreos-installer download --decompress -s "${STREAM}" -p vmware -f vhd.xz
+----
+
+Alternatively, you can manually download an Azure image from the https://getfedora.org/coreos/download?tab=cloud_operators[download page].  Verify the download, following the instructions on that page, and decompress it.
+
+== Uploading the image to Azure
+
+. Create any resources that don't already exist in your Azure account:
++
+.Example creating Azure resources
+[source, bash]
+----
+# Create resource group
+az group create -n my-group -l westus2
+# Create storage account for uploading FCOS image
+az storage account create -n mystorageacct -g my-group
+# Retrieve connection string for storage account
+cs=$(az storage account show-connection-string -n mystorageacct | jq -r .connectionString)
+# Create storage container for uploading FCOS image
+az storage container create --connection-string "$cs" -n my-container
+----
+
+. Create an FCOS image:
++
+.Example creating Azure image
+[source, bash]
+----
+# Upload image blob
+az storage blob upload --connection-string "$cs" -c my-container -f <downloaded-image-file.vhd> -n my-fcos-image.vhd
```

Comment:
Since we used variables in other places maybe we should make one here:

```
file=<downloaded-image-file.vhd>
az storage blob upload --connection-string "$cs" -c my-container -f "$file" -n my-fcos-image.vhd
```